frontend/pages/weights/[slug].tsx

```diff
@@ -0,0 +1,86 @@
+import { GetStaticPaths, GetStaticProps, InferGetServerSidePropsType } from "next";
+import Head from "next/head";
+import Image from "next/image";
+import { Headline } from "../../components/Headline/Headline";
+import { Tag } from "../../components/Tag/Tag";
+
+// TODO (Zoe-Bot): Adjust types when list page merged and when using weights of course xD
+type Todo = {
+    userId: number
+    id: number
+    title: string
+    completed: boolean
+}
+
+// TODO (Zoe-Bot): Adjust types when list page merged
+type WeightsSingleProps = {
+    item: Todo
+}
+
+/** Single Page of a weight */
+export default function WeightsSingle({ item }: InferGetServerSidePropsType<typeof getStaticProps>) {
+    const siteTitle = `How much does ${item.title} weight | WWWeights`
```

Comment:
```suggestion
    const siteTitle = `${item.title} Weight | WWWeights`
```

Die Frage geht nicht ohne algo zu checken ob artikel stimmt. Machen wir erstmal so.